sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAsyncClient.java

```diff
@@ -189,13 +185,13 @@ HttpPipeline getHttpPipeline() {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<IndexDocumentsResult>> uploadDocumentsWithResponse(Iterable<?> documents,
-        IndexDocumentsOptions options, RequestOptions requestOptions) {
-        return withContext(context -> uploadDocumentsWithResponse(documents, options, requestOptions, context));
+        IndexDocumentsOptions options) {
```

Comment:
Remember to remove the parameters

Reply:
Not following completely on this request. Which parameters need to be removed?

Reply:
Sorry, I mean the JAVADOC